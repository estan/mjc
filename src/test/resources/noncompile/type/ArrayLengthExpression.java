class ArrayLengthExpressionTest {
    public static void main(String[] args) {
        int i;
        int[] ia;
        boolean b;
        A cA;
        B cB;

        i = i.length;  // LENGTH_ON_NON_ARRAY
        //i = ia.length;  // OK!
        i = b.length;  // LENGTH_ON_NON_ARRAY
        i = cA.length; // LENGTH_ON_NON_ARRAY
    }
}
class A {}
class B {}
