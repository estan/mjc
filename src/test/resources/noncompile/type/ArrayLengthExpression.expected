LENGTH_ON_NON_ARRAY
LENGTH_ON_NON_ARRAY
LENGTH_ON_NON_ARRAY
